bin/environment.js

```diff
@@ -140,10 +140,6 @@ exports.bootstrap = (eg, adapter) => {
     .demandCommand()
     .recommendCommands()
     .strict()
-    .alias('h', 'help')
```

Comment:
What's the impact of this block of deletions?

Reply:
According to the `yargs` changelog, `--help, -h and --version` are now automatically added. It was a boilerplate for everybody and they decided to put it as default.

Reply:
Oh, I just saw that help and version are included by default.  However, I don't think `-h` is aliased by default, so we may need that.

Reply:
That's true. However I also realised that alias is automatically added [here](https://github.com/ExpressGateway/express-gateway/blob/master/bin/eg-generator.js#L60)

![image](https://user-images.githubusercontent.com/1416224/37369538-ab3416fe-2709-11e8-9af7-dc046cd176ab.png)

Reply:
Just tried locally.  I get this:

```
➜ ./bin/index.js -h    
<snip>

Not enough non-option arguments: got 0, need at least 1
```

`--help` works, though.

Reply:
The output on my machine is the same. Yargs is complaining of the "non-option" arguments.

![image](https://user-images.githubusercontent.com/1416224/37369819-850e9624-270a-11e8-9de1-24eabd379e41.png)